example_scripts/example_runScripts.py

```diff
@@ -12,64 +12,51 @@
 
 from fitbenchmarking.fitting_benchmarking import fitbenchmark_group
 from fitbenchmarking.results_output import save_results_tables
+from fitbenchmarking.utils.options import Options
 
 
 def main(args):
-    # SPECIFY THE SOFTWARE/PACKAGE CONTAINING THE MINIMIZERS YOU WANT TO BENCHMARK
-    software = ['scipy', 'sasview']
-    software_options = {'software': software}
-
-    # User defined minimizers
-    # custom_minimizers = {"scipy": ["lm-scipy", "trf", "dogbox"],
-    #                      "sasview": ["amoeba", "lm-bumps", "newton", "de", "pt", "mp"]}
-    # None will default to the list on the options file
-    custom_minimizers = None
-
-    # SPECIFY THE MINIMIZERS YOU WANT TO BENCHMARK, AND AS A MINIMUM FOR THE SOFTWARE YOU SPECIFIED ABOVE
+    # Find the options file
     current_path = os.path.dirname(os.path.realpath(__file__))
     if len(args) > 1:
         # Read custom minimizer options from file
-        software_options['minimizer_options'] = None
-        software_options['options_file'] = current_path + args[1]
```

Comment:
Shall we use argparse? Could be a different way to add options down the line too

Reply:
I think that should be a separate work package